frontend/webservice/monitoring.cgi

```diff
@@ -248,5 +258,48 @@ sub send_json {
     print "Content-type: text/plain\n\n" . encode_json($output);
 }
 
+sub get_oess_status {
+    my $result;
+    my $serviceResult;
+    
+    return if !defined $mq;
+
+    $mq->{'topic'} = 'MPLS.Discovery.RPC';
+    $mq->{'timeout'} = 2;
+    $serviceResult = $mq->is_online();
+    if (defined $serviceResult->{error}) {
+        push(@$result,{ServiceName => 'Discovery', Status => 'Down'});
+    } else {
+        push(@$result,{ServiceName => 'Discovery', Status => 'Up'});
+    }
+    
+    $mq->{'topic'} = 'MPLS.FWDCTL.RPC';
+    $serviceResult = $mq->echo();
+    if (defined $serviceResult->{error}) {
+        push(@$result,{ServiceName => 'FWDCTL', Status => 'Down'});
+    } else {
+        push(@$result,{ServiceName => 'FWDCTL', Status => 'Up'});
+    } 
+     
+     $mq->{'topic'} = 'OF.FWDCTL.RPC';
+     $serviceResult = $mq->is_online_notif();
```

Comment:
I think we can probably come up with a better name for this function but it's not a blocker.

Reply:
replaced with is_online() to keep consistency. Not an issue since FWDCTL uses the MPLS.FWDCTL.RPC and not the OF.FWDCTL.RPC. We could still go back through and rename all the functions if desired but for the time being they are consistent